src/main/java/duke/TodoCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.io.IOException;
+
+public class TodoCommand extends Command {
+    private String description;
+
+    /**
+     * Constructor for a Todo command.
+     *
+     * @param description Description of the todo task.
+     */
+    public TodoCommand(String description) {
+        this.description = description;
```

Comment:
Similarly to the other Task commands, just create the object in the constructor itself.